### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by danielmao2019** [cjf8899/CDRL#9]
According to the paper the loss L_gan should have `log` operations in the computation but I did not see any `log` function used in the training code in `main.py`?
